FileTypeSupport.MXLIFF/MXLIFFWriter.cs

```diff
@@ -149,50 +146,41 @@ private void AddComments(XmlNode xmlUnit, List<IComment> comments)
                 text += c.Text + " ";
             }
 
-            var createdat = targetFile.CreateAttribute("created-at");
-            var createdby = targetFile.CreateAttribute("created-by");
-            var modifiedat = targetFile.CreateAttribute("modified-at");
+            var createdat = _targetFile.CreateAttribute("created-at");
```

Comment:
Is it possible that _targetFile value is null?

Reply:
Is instanciated in line 63.